src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -176,49 +171,42 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         /// The callback function that is invoked when the message is received from the script.
         /// </summary>
         [UnmanagedFunctionPointer(CallingConvention.StdCall)]
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public delegate void JavaScriptMessageHandler(string message);
 
         /// <summary>
         /// The callback function that is invoked when the message is received from the script.
         /// </summary>
         [UnmanagedFunctionPointer(CallingConvention.StdCall)]
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public delegate void JavaScriptAlertCallback(string message);
```

Comment:
I did not pay attention to JavaScriptMessageHandler API before.
It looks JavaScriptMessageHandler does the same thing as XXXCallback.
In order to make code consistent, there are 2 options:
1) Change XXXCallback to XXXHandler, while there is a class name with Handler like WebHttpAuthHandler. 
2) Change JavaScriptMessageHandler to JavaScriptMessageCallback.
@rabbitfor  What do you think?